### PR TITLE
python3-libtmux: update to 0.8.3

### PR DESCRIPTION
**Pull request by egorenar**
Signed-off-by: Alexander Egorenkov <egorenar-dev@posteo.net>

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by egorenar** [comment]
This change is necessary for PR #27876

**Review by ericonr — COMMENTED** [review]
Please address these issues in #27876. Since the updates are linked, they should happen in a single PR (with one commit per package).

I will be closing this PR, and you can update everything in #27876. Feel free to ping me there once you're done.

File: srcpkgs/python3-libtmux/template [hunk 1]
```diff
@@ -1,17 +1,18 @@
 # Template file for 'python3-libtmux'
 pkgname=python3-libtmux
-version=0.8.1
+version=0.8.3
```
[the remark this answers is not in the document]
Comment (egorenar) [reply srcpkgs/python3-libtmux/template:3]: I'm confused. I see only 0.8.3 as latest release. I missed something ?

[the remark this answers is not in the document]
Comment (ericonr) [reply srcpkgs/python3-libtmux/template:3]: Hm, I think I looked at changelog.

[the remark this answers is not in the document]
Comment (egorenar) [reply srcpkgs/python3-libtmux/template:3]: Yeah, it's odd, there is no 0.8.5 release on GitHub but there is a tag :) Anyways i fixed it and it seems to work.
I moved the change to #27876.
Thanks for feedback.

**Pull request closed without merging**